From 7a77e4df94a48c35f9a4bf1fc3f8e9d1f72a77b7 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Wed, 6 Feb 2019 14:50:24 +0100
Subject: [PATCH] tc: include stdint.h explicitly for UINT16_MAX

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1641909
Upstream Status: iproute2.git commit ae717baf15fb4

commit ae717baf15fb4d30749ada3948d9445892bac239
Author: Khem Raj <raj.khem@gmail.com>
Date:   Sat May 20 14:28:46 2017 -0700

    tc: include stdint.h explicitly for UINT16_MAX

    Fixes
    | tc_core.c:190:29: error: 'UINT16_MAX' undeclared (first use in this function); did you mean '__INT16_MAX__'?
    |    if ((sz >> s->size_log) > UINT16_MAX) {
    |                              ^~~~~~~~~~

    Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 tc/tc_core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tc/tc_core.c b/tc/tc_core.c
index 7bbe0d7..821b741 100644
--- a/tc/tc_core.c
+++ b/tc/tc_core.c
@@ -12,6 +12,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdint.h>
 #include <unistd.h>
 #include <syslog.h>
 #include <fcntl.h>
-- 
1.8.3.1

